SwiftUIFourierSeries/Views/ConfigView.swift

```diff
@@ -3,46 +3,52 @@ import SwiftUI
 
 struct ConfigView: View {
     @ObservedObject var store: Store
+    @Environment(\.verticalSizeClass) var verticalSizeClass: UserInterfaceSizeClass?
+    @Environment(\.horizontalSizeClass) var horizontalSizeClass: UserInterfaceSizeClass?
     
     var body: some View {
-        HStack {
-            VStack {
-                HStack {
-                    if self.store.state.timerIsRunning {
-                        Button("Pause") { self.store.send(.pause) }
-                    } else {
-                        Button("Play") { self.store.send(.play) }
+        // https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/
+        Group {
```

Comment:
I don't think we need to wrap this in a `Group`. It seems to look fine without it.